<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCV_Grab_dualClass</class>
 <widget class="QMainWindow" name="OpenCV_Grab_dualClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV_Grab_dual</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,9">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>相机1前缀名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>相机2前缀名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>去畸变</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="MyLabel" name="img1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="img2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionStop"/>
   <addaction name="actionRecord"/>
   <addaction name="actionSettings"/>
   <addaction name="actionColor"/>
   <addaction name="actionExplorer"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/Start.ico</normaloff>:/OpenCV_Grab_dual/Resources/Start.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/Stop.ico</normaloff>:/OpenCV_Grab_dual/Resources/Stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/Save.ico</normaloff>:/OpenCV_Grab_dual/Resources/Save.ico</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/Setting.ico</normaloff>:/OpenCV_Grab_dual/Resources/Setting.ico</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/cai.ico</normaloff>
     <normalon>:/OpenCV_Grab_dual/Resources/gray.ico</normalon>:/OpenCV_Grab_dual/Resources/cai.ico</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionExplorer">
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/Explorer.ico</normaloff>:/OpenCV_Grab_dual/Resources/Explorer.ico</iconset>
   </property>
   <property name="text">
    <string>Explorer</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="OpenCV_Grab_dual.qrc">
     <normaloff>:/OpenCV_Grab_dual/Resources/full_garbage.ico</normaloff>
     <normalon>:/OpenCV_Grab_dual/Resources/bin_empty.ico</normalon>:/OpenCV_Grab_dual/Resources/full_garbage.ico</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="OpenCV_Grab_dual.qrc"/>
 </resources>
 <connections/>
</ui>
